sdv/data_processing/data_processor.py

```diff
@@ -408,15 +496,24 @@ def create_regex_generator(self, column_name, sdtype, column_metadata, is_numeri
         return transformer
 
     def _get_transformer_instance(self, sdtype, column_metadata):
+        transformer = self._transformers_by_sdtype[sdtype]
+        if transformer.__class__.__name__ == 'AnonymizedFaker':
```

Comment:
We have `AnonymizedFaker` imported, so we can just check if its instance of it.
`if isinstance(transformer, AnonymizedFaker)`